app/managing-users.md

```diff
@@ -55,32 +55,32 @@ Okta allows new users to sign in and use RAVS.
 
-RAVS cannot onboard users to multiple organisations with the same email address. Only one email can be linked to one organisation. 
+The user will then be able to use the same email address to access RAVS at each organisation. 
 
-If you encounter an ‘email already exists’ error message while adding a user, their email is already linked to another organisation. To add this user to your organisation, you can use a variation of their email by including `+organisationname` to their email address.  
+Users can have different permission levels at each organisation.
 
-You must do this to help users identify which account they’re logging into. Not doing so, could lead to users recording vaccinations under an incorrect organisation.  
+## If a user was added to multiple organisations before 16 December 2024
 
-For example, Jane works across multiple organisations. Her primary email jane.carlton2@nhs.net is linked to Hollybrook hospital.  
+Before 16 December 2024, to access RAVS at multiple organisations, a user needed a separate email address linked to each organisation.
 
-If you’re a lead user at: 
+If a user has more than 1 email address to access RAVS, they can continue to log in as before, using separate emails for each organisation.
 
-* Greenwood pharmacy, add Jane to your organisation by entering jane.carlton2+greenwoodpharmacy@nhs.net 
-* Rosebush hospital, add Jane to your organisation by entering jane.carlton2+rosebushhospital@nhs.net
+Or, if they want to only use their main email address to access RAVS at each organisation, they can do the following:
 
-You cannot add users to other organisations.  
+* add themselves (or ask a lead admin to add them) to an organisation using their main email address
+* then deactivate (or ask a lead admin to deactivate) the extra email account they used previously for that organisation
 
-### What happens next? 
+Any records previously created with an extra email account will remain unchanged. 
 
-* Each email will have its own OKTA account and is linked to one organisation.  
-* When you add users to your organisation with a variation of their email, we’ll automatically send a RAVS and Okta welcome email. Emails are sent to users’ primary email inboxes with instructions to activate their accounts.  
-* Users must log in with the email linked to the organisation where they’re delivering vaccinations to maintain accurate clinical records. 
+## If a user has been added to multiple organisations after 16 December 2024
 
-**Note**: In future, you'll be able to add users to your organisation even if their email is linked to another. This means when users log in they can switch between organisations. When we introduce this change, all varying email accounts will be merged into users’ primary emails.   
+Each time they log in, they will see a screen listing all the organisations they’ve been added to. They then have to select the relevant organisation.
```

Comment:
Does this need to be tweaked to reflect that they’ll not see any organisations they’ve been deactivated from, or is this obvious enough to be assumed?

```suggestion
Each time they log in, they will see a screen listing all the organisations they’re currently associated with. They then have to select the relevant organisation.
```